azure-media/src/main/java/com/microsoft/windowsazure/services/media/authentication/AzureAdClientUsernamePassword.java

```diff
@@ -26,8 +26,8 @@ public String getPassword() {
 
     /**
      * Initializes a new instance of the AzureAdClientSymmetricKey class.
-     * @param clientId The client ID.
-     * @param clientKey The client key.
+     * @param username The client username
```

Comment:
Not sure I agree with this change, as it is not a username - it is a Service Principal AD application client ID and Key

Reply:
The problem with 'fixing' this code by inverting what I did - e.g. changing the argument names into the constructor to be `clientId` and `clientKey` is that the class has two methods  - `getUsername()` and `getPassword()`, so fixing in one place as you suggest will indeed fix the JavaDoc issues, but it will not necessarily fix the semantics based on your comment. Thoughts?

Reply:
I'm wondering if there is confusion between the AzureAdClientSymmetricKey class and the one above - the AzureAdClientUsernamePassword class - perhaps the one above should have username and password as the parameter names (to match the get methods)?

Reply:
@johndeu Are you able to comment on this so that I may try to fix this issue? Thanks!